GIS.md

```diff
@@ -0,0 +1,198 @@
+
+To accommodate this workflow Ubuntu ships with Gnome Initial Setup, a program that can safely handle user account 
+creation and configuration. Gnome Initial Setup is started by default when a system is booted and GDM is unable to
+detect any user accounts. Instead of taking you to the login screen, a special Gnome Initial Setup user session is started.
+
+This guide aims to highlight some of the ways a system can be provisioned such that Ubuntu is installed, but no user 
+account is created, allowing user account creation to be passed off to Gnome-Initial-Setup.
+
+# Triggering Gnome-Initial-Setup via the new Flutter installer and a whitelabel.yaml
+
+The new [Ubuntu Desktop Bootstrap](https://github.com/canonical/ubuntu-desktop-provision) Flutter installer can be 
+given a `whitelabel.yaml` file to customize its appearance and behavior. One thing you can configure is the `mode` you 
+would like to run the installer in. 
+
+The `oem` mode skips user creation and time zone setup during installation, leaving them to be handled by Gnome Initial 
+Setup post installation.
+
+The following `whitelabel.yaml` placed in `/usr/share/desktop-provision/whitelabel.yaml` on your LiveCD is sufficient 
+to skip user creation during installation, provided you are using the new Ubuntu Desktop Bootstrap installer:
+
+```
+mode: oem
+```
+
+A comprehensive guide to customizing a LiveCD can be found [here](https://help.ubuntu.com/community/LiveCDCustomization).
+
+## Triggering Gnome-Initial-Setup without modifying the LiveCD
+
+It's possible to test the Gnome Initial Setup user creation flow via the Ubuntu Desktop Bootstrap installer without first 
+editing your LiveCD, provided Gnome Initial Setup is already present on your ISO.
+
+### Install ubuntu-desktop-bootstrap
+
+The following command installs the Ubuntu Desktop Bootstrap Flutter installer
+```
+sudo snap install ubuntu-desktop-bootstrap --classic
+```
+
+### Write a whitelabel.yaml to set OEM mode
+
+These commands create a `whitelabel.yaml` in the location Ubuntu Desktop Bootstrap will look for them.
+```
+sudo mkdir -p /usr/share/desktop-provision
+
+sudo bash -c 'echo "mode: oem" > /usr/share/desktop-provision/whitelabel.yaml'
+```
+
+### Launch ubuntu-desktop-bootstrap
+
+Finally, launch the installer. You should be no be prompted for user account creation, which will be handled after the 
+installer finishes
+```
+/snap/bin/ubuntu-desktop-bootstrap --try-or-install
+```
+
+# Triggering Gnome-Initial-Setup via an autoinstall.yaml
+
+These sections are adapted from the subiquity [documentation](https://canonical-subiquity.readthedocs-hosted.com/en/latest/howto/autoinstall-quickstart.html), updated for triggering Gnome Initial Setup with the Ubuntu Desktop ISO
+
+## Using an autoinstall provided over the network
+
+### Getting an ISO
+You can find a desktop ISO to download on the [official Ubuntu website](https://ubuntu.com/download/desktop). (Currently the latest is Ubuntu 24.04 LTS)
```

Comment:
Not sure we want to call out 24.04 as the latest version here? We'd need to keep that up to date each time we had a release :slightly_smiling_face: